util/pace/util/_legacy_restart.py

```diff
@@ -24,6 +23,7 @@ def open_restart(
     only_names: Iterable[str] = None,
     to_state: dict = None,
     tracer_properties: RestartProperties = None,
+    fortran_restart: bool = False,
```

Comment:
What is the purpose of this argument? This function is specifically only for loading Fortran restarts (hence the filename).

Reply:
Removed